po/glossary/pt_BR.po

```diff
@@ -72,139 +72,139 @@ msgstr "tipo da conta: moeda"
 
 #. "see: Equity, but see also 'account type: Passive'"
 msgid "account type: Equity"
-msgstr "Patrimônio Líquido"
+msgstr "tipo da conta: Patrimônio Líquido"
```

Comment:
@fellen in Brazilian Portuguese "behind" means "at your back", do you mean "translate only the part to the right of the colon" and ignore everything that comes before it?
It will be fixed.